projects/showcase/src/app/datasets/blood-pressure-mapper.ts

```diff
@@ -38,12 +38,15 @@ export class BloodPressureMapper implements Mapper<BloodPressureObservation> {
       //   yMin: 80,
       // }),
       merge({}, this.annotationOptions, {
+        display: true,
```

Comment:
is `display: true` necessary?

Reply:
yes, colin it is necessary. we are using that display property in the checkbox checked, I have tested by removing this but then it not working properly.